README.md

```diff
@@ -2,12 +2,14 @@
 
 Durante o mês de outubro, o Hacktoberfest convida a todos para contribuir para esta comunidade global. Todos os backgrounds e níveis de habilidade em desenvolvimento de projetos open source são bem-vindos. 
 
-Participando do evento, você ajuda no crescimento de projetos open source e aprende com a mentoria de contribuidores de longa data. E mais: se você concluir o desafio do evento, você pode ganhar a edição limitada da camiseta do Hacktoberfest de 2020.
+Participando do evento, você ajuda no crescimento de projetos open source e aprende com a mentoria de contribuidores de longa data. E mais: se você concluir o desafio do evento, você pode ganhar a edição limitada da camiseta do Hacktoberfest de 2020 (ou plantar uma árvore!!).
 
 Com o objetivo de facilitar o acesso aos participantes do evento (sejam da nossa empresa ou não), a Quero Educação criou este repositório para centralizar as informações e disponibilizar mentoria a outros participantes.
 
 ## Dicas e Artigos sobre como contribuir com o Open Source
 
+- Procure por tags `"good first issue"` nos repositórios públicos, essas são issues que os projetos marcaram como uma boa entrada ao projeto pelo nível de díficuldade de entrada dela, geralmente exigindo menos conhecimento prévio sobre o projeto, é um bom primeiro passo :muscle: outras tags legais pra procurar são: `"help-wanted"`, `"documentation"` e `"contributions welcome"`, porém essas duas ultimas podem não ser tão simples :smile:;
+
 // TODO
```

Comment:
Pode remover o TODO?

Reply:
só com uma dica? ahahah

`// TODO: Adicionar mais`

Reply:
Sem querer ser o chato, mas meio que já sendo, teria como colocar acento em últimas (última frase)?

Reply:
```diff
- porém essas duas ultimas
+ porém essas duas primeiras
```

Reply:
> Sem querer ser o chato, mas meio que já sendo, teria como colocar acento em últimas (última frase)?

poxa, como assim "ser chato", perdão eu, por ser ~analfabéto~ desatento. :sweat_smile: 

> ```diff
> - porém essas duas ultimas
> + porém as duas primeiras
> ```

na verdade essas três ultimas podem não ser tão newby friendly, né? cê acha melhor tirar essa ultima parte (ficar só até o `:muscle:`)?

Reply:
É que tem gente que não gosto de ter o português corrigido, =P.

Melhor manter a frase, para separar o `good first issue` dessas últimas três

Reply:
A última acho que pode envolver PR's em outro repositório como dependência, algo como `wontfix`. Então, talvez seja melhor trocar por `porém essas são ...'. Oque acham?